app/views/content_items/service_sign_in/_choose_sign_in.html.erb

```diff
@@ -27,6 +27,7 @@
   <%= render "govuk_publishing_components/components/fieldset", legend_text: legend_text do %>
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
+      <div id="verify-hint" data-module="show-gov-uk-verify-hint" style="display:none"></div>
```

Comment:
We should avoid in-line styling. Can we rely on CSS and `.js-enabled` class to hide this?
Also I don't see where we need an `id` for this element.

Reply:
As much as I hate in-inline styling I have resorted to this as I don't want to make the element (even empty) visible. It's not only for people with no-js (so `.js-enabled` class does not help) but people who haven't used Verify successfully before shouldn't have get the element visible... 

The ID is a leftover from original implentation, it didn't get used. I'll remove it.